app/models/service_template.rb

```diff
@@ -63,6 +63,7 @@ class ServiceTemplate < ApplicationRecord
 
   belongs_to :service_template_catalog
   belongs_to :zone
+  belongs_to :currency, :class_name => "ChargebackRateDetailCurrency", :inverse_of => false
```

Comment:
What does the `:inverse_of => false` do here?

Reply:
Ah, @jrafanie found https://github.com/rails/rails/blob/5-0-stable/activerecord/lib/active_record/associations.rb#L704-L718 for me so it sounds like this is just telling AR that we're never going to ask for a currency's service templates which makes sense.